src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java

```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```

Comment:
the same compilation error for java8 and java7 , when I move that code to separate stanalone class to compile it by "javac" it works fine.

```
+import java.lang.Class;
+

```
that is workaround for a problem, but importing "java.lang" most likely just another violation in Idea.